apps/condo/domains/notification/constants/constants.js

```diff
@@ -31,9 +34,21 @@ const MESSAGE_TYPES = [
     SHARE_TICKET_MESSAGE_TYPE,
     DEVELOPER_IMPORTANT_NOTE_TYPE,
     CUSTOMER_IMPORTANT_NOTE_TYPE,
-    MESSAGE_FORWARDED_TO_SUPPORT,
+    MESSAGE_FORWARDED_TO_SUPPORT_TYPE,
+    TICKET_ASSIGNEE_CONNECTED_TYPE,
+    TICKET_EXECUTOR_CONNECTED_TYPE,
+    TICKET_STATUS_CHANGED_TYPE,
 ]
 
+const DEFAULT_TRANSPORT_PRIORITIES = [SMS_TRANSPORT, EMAIL_TRANSPORT]
+const PUSH_EMAIL = [PUSH_TRANSPORT, EMAIL_TRANSPORT]
```

Comment:
If it also priorities, rename the constant, please

Reply:
done